aws/aws-how-to/instances/upgrade-to-ubuntupro-with-license-manager.rst

```diff
@@ -0,0 +1,50 @@
+Upgrade Ubuntu instances to Ubuntu Pro with AWS License Manager
+===============================================================
+
+
+AWS released a new functionality that allows users upgrade Ubuntu LTS instances to `Ubuntu Pro`_ with `AWS License Manager`_.
+
+
+Requirements
+------------
+
+To upgrade to Ubuntu Pro, you will need
+
+* Ubuntu LTS EC2 machines 16.04 or later LTS versions
+* EC2 machines managed by `AWS Systems Manager`_ (SSM)
+    * Follow `AWS Official SSM Manager document`_, `Enabling AWS SSM video tutorial`_ or `step-by-step tutorial`_ to create IAM role and access instances with SSM
+    * To check if instances are managed by SSM, go to the `SSM console`_ and check if the instances to be upgraded appear in :guilabel:`Systems Manager` > :guilabel:`Fleet Manager` > :guilabel:`Managed nodes`.
+    * You may need to either restart the instance or directly the SSM agent by running ``sudo snap restart amazon-ssm-agent`` to reactivate
+* Install latest pro client tool by running ``sudo apt install ubuntu-advantage-tools``
+* Check with `network requirements`_ and `proxy configuration`_ for Ubuntu Pro to give network access to instances.
+
+
+Upgrade to Ubuntu Pro
+---------------------
+
+Once you checked all requirements in previous section, go to `EC2 dashboard`_ and shut down all the instances you would like to upgrade. ``Managed nodes`` will look like below.
```

Comment:
isn't there an option for an upgrade without downtime? if not, I think we should mention that here.

Reply:
You can select instances if you don't stop your instances. I fixed `shut down` -> `stop` and added it will not be shown if instances are running.